versions/1/changelog.yml

```diff
@@ -248,4 +251,4 @@
     link: https://github.com/elastic/package-spec/pull/193
   - description: Introduce "value" property for constant_keyword fields
     type: enhancement
-    link: https://github.com/elastic/package-spec/pull/194
+    link: https://github.com/elastic/package-spec/pull/194
```

Comment:
Hm.. it looks like you removed a new line here.

Reply:
oops. Fixed.